libs/service-portal/core/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react'
+import * as styles from './Modal.treat'
+import { Box, IconDeprecated as Icon } from '@island.is/island-ui/core'
+
+interface Props {
+  isOpen: boolean
+  onCloseModal: () => void
+}
+
+export const Modal: FC<Props> = ({ isOpen, children, onCloseModal }) => {
```

Comment:
You are missing some a11y features, right?
https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html

We have Reakit as a global dependency that could be a good starter for modal accessibility:
https://reakit.io/docs/dialog/

We need a modal in `island-ui`. Can I use yours as a base, improve a11y and publish under `island-ui`?

Reply:
Yeah didn't really go into details since I knew we would be implementing one for the UI library

Of course you can use it as a base :)